server/graphql/programs/src/types/encounter.rs

```diff
@@ -71,23 +71,27 @@ impl EncounterEventFilterInput {
             document_type: None,
             document_name: None,
             r#type: self.r#type.clone().map(EqualFilter::from),
-            context: None,
+            context_id: None,
         }
     }
 }
 
 #[Object]
 impl EncounterNode {
     pub async fn id(&self) -> &str {
-        &self.encounter_row.id
+        &self.encounter.0.id
```

Comment:
just a whine, rather than a comment: I have to say that I find the tuples much harder to read. How do I know that `encounter.0` is an `EncounterRow` just by reading the line?

Reply:
Agree, not optimal. I will leave for now but whay I will do next time is something like: `self.encounter_row().id`.